lab/components/AsyncInlineSvg.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+	<!--
+		hack I temporarily used to fix an issue
+		that seems to have magically fixed itself
+		but I'm keeping this code here in case
+		I wanna experiment more with it later
+	-->
+	<html-fragment
+		v-if="fragment"
+		:class="classes"
+		:html="svgText"
+	/>
+	<!-- eslint-disable vue/no-v-html -->
+	<i
```

Comment:
Why the italic tag over, say, a span?

Reply:
i thought the `i` in `<i>` stood for `icon`, oops, 😆 .

i think i was mislead to have this belief by Font Awesome. All of their docs recommend using `<i>` but i honestly have no clue why. I'm down to change this to a `<span>` 👍